### ISSUE TITLE
VerticalNavigation missing aria-current attribute for the selected item

### ISSUE THREAD

**Issue by Prophetess** [salesforce/design-system-react#2747]
Referring to the [SLDS component blueprint](https://core-main.lightningdesignsystem.com/components/vertical-navigation/#site-main-content) for the vertical navigation, the attribute `aria-current` should be used to indicate the selected item.
